galaxy/values.yaml

```diff
@@ -132,6 +132,24 @@ extraVolumeMounts: []
 
 initJob:
   createDatabase: true
+  externalConfs:
+    enabled: true
+    # Uses Galaxy's shared file system volume
+    volume:
+      subPath: cvmfsclone # on galaxy-data volume (avoid config, tools, lib, etc...)
+      mountPath: /cvmfs/cloud.galaxyproject.org
+    archives:
+      # A tar.gz publicly accessible archive containing AT LEAST conf files and XML tool wrappers.
+      # Meant to be enough for Galaxy handlers to startup
+      startup: https://storage.googleapis.com/cloud-cvmfs/startup.tar.gz
```

Comment:
Should we simplify this by accepting a url to the archive instead (instead of having 3 settings named startup, running and full)?

Reply:
I liked the three so that we are able to optimize startup, but leave the option for someone else to just tar up all their confs and use a single full tar everywhere. So in the view of the chart there are always three different stages of decompressing, but in reality someone could just put the full one for all three settings, hence the comment that each tar needs to containt _at least_ a certain potion.